src/SFA.DAS.AssessorService.Application.Api/Services/OrganisationMergingService.cs

```diff
@@ -0,0 +1,280 @@
+                Status = sourceOrganisationStandardVersion.Status,
+                Comments = sourceOrganisationStandardVersion.Comments,
+                DateVersionApproved = sourceOrganisationStandardVersion.DateVersionApproved,
+                EffectiveFrom = sourceOrganisationStandardVersion.EffectiveFrom,
+                EffectiveTo = sourceOrganisationStandardVersion.EffectiveTo,
+            };
+
+            return mosv;
+        }
+        private MergeOrganisationStandardDeliveryArea CreateMergeOrganisationStandardDeliveryArea(string replicates, OrganisationStandardDeliveryArea sourceOrganisationStandardDeliveryArea)
+        {
+            var mosda = new MergeOrganisationStandardDeliveryArea()
+            {
+                Replicates = replicates,
+                DeliveryAreaId = sourceOrganisationStandardDeliveryArea.DeliveryAreaId,
+                OrganisationStandardId = sourceOrganisationStandardDeliveryArea.OrganisationStandardId,
+                Status = sourceOrganisationStandardDeliveryArea.Status,
+                Comments = sourceOrganisationStandardDeliveryArea.Comments,
+            };
+
+            return mosda;
+        }
+
+
+        private void MergeOrganisationStandardsAndVersions(Organisation primaryOrganisation, Organisation secondaryOrganisation, Guid createdByUserId, DateTime secondaryStandardsEffectiveTo)
+        {
+            // @ToDo: a bit monolithic - can this be refactored in to something more readable?
+
+            // Grab the primary contact for the primary organisation because we're going to need it.
+            var primaryContact = _dbContext.Contacts.FirstOrDefault(c => c.EndPointAssessorOrganisationId == primaryOrganisation.EndPointAssessorOrganisationId && c.Email == primaryOrganisation.PrimaryContact);
+
+            // Read all the standards from the secondary organisation.
+
+            foreach(var secondaryOrganisationStandard in secondaryOrganisation.OrganisationStandards)
+            {
+                // Does the primary organisation have this standard already?
+                var primaryOrganisationStandard = primaryOrganisation.OrganisationStandards.FirstOrDefault(pos => pos.StandardCode == secondaryOrganisationStandard.StandardCode);
+
+                if (null == primaryOrganisationStandard)
+                {
+                    // No - so add the standard to the primary organisation.
+
+                    primaryOrganisationStandard = new OrganisationStandard()
+                    {
+                        DateStandardApprovedOnRegister = secondaryOrganisationStandard.DateStandardApprovedOnRegister,
+                        OrganisationStandardData = secondaryOrganisationStandard.OrganisationStandardData,
+                        Comments = $"{secondaryOrganisationStandard.Comments} ** this standard has been merged from Organisation {secondaryOrganisation.EndPointAssessorOrganisationId}",
+                        ContactId = primaryContact?.Id,
```

Comment:
I'm also back to whether you would want to put some sort of `Clone` method on the OrganisationStandard entity to allow you to abstract the logic out of this merge service, then can test against that specifically via unit tests to break this down. Appreciate we have to do some form of copy as we don't want to copy across the EF context tracking.